fe/fe-core/src/main/java/com/starrocks/sql/optimizer/transformer/CTETransformerContext.java

```diff
@@ -0,0 +1,83 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.sql.optimizer.transformer;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.Maps;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class CTETransformerContext {
+    private final Map<Integer, ExpressionMapping> cteExpressions;
+
+    // cteMould -> current cte ref
```

Comment:
what's the meaning of `cteMould `?

Reply:
For discriminate cteID，means every instance of CTE is copy from this mould, would you give a better name?